configs/agent_config.go

```diff
@@ -53,6 +55,72 @@ type AgentHooks struct {
 	DoOnWorkflowEnd   string `yaml:"do_on_workflow_end"`
 }
 
+func RegisterAgentOverrides() error {
+	if !hasAgentConfigFile() {
+		return nil
+	}
+
+	file := filepath.Join(GetBitriseHomeDirPath(), agentConfigFileName)
+
+	log.Print("")
+	log.Info("Running in agent mode")
+	log.Printf("Config file: %s", file)
+
+	config, err := readAgentConfig(file)
+	if err != nil {
+		return fmt.Errorf("agent config file: %w", err)
+	}
+	log.Printf("env: %s", os.Getenv(BitriseDataHomeDirEnvKey))
```

Comment:
Is this a leftover print statement? Just asking because when it is executed it will print out something like
```
env: /path/to/some/folder
```
and if the folder does not have a clear name then (without knowing the CLI codebase) it will be hard to understand the users what env is being printed out. 

So I think we should either add the env name into the print statement or remove it.